library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java

```diff
@@ -160,20 +160,28 @@ String resolveTemplate(final String templateYAML, final String platformFilterTem
 
     String dnsFallbackNameserversAsString = "[]";
     if (!dnsFallbackNameservers.isEmpty()) {
-      StringJoiner sj = new StringJoiner(",", "[", "]");
+      StringBuilder sb = new StringBuilder("[");
```

Comment:
Can we use `TextUtils` instead? https://developer.android.com/reference/android/text/TextUtils#join(java.lang.CharSequence,%20java.lang.Object[])

Reply:
This package doesn't currently depend on android packages:

```
library/java/io/envoyproxy/envoymobile/engine/EnvoyConfiguration.java:3: error: package android.text does not exist
import android.text.TextUtils;
                   ^
```